### DISCUSSION TITLE
Expired trust anchor

### DISCUSSION THREAD

**Discussion by hopik007** [linkerd/linkerd2#5071] [General]
Hi
I am reading doc about manual rotate  of trust anchor.
Could anobody tell me please what happened with k8s when this trust anchor expire ?
If everything will stop working or something else.

Thanks a lot

**Comment by cpretzer** [comment]
hi @hopik007 

Kubernetes functionality won't be affected when the trust anchor expires. The trust anchor is the underlying source of truth in the chain that is used to generate the certificates that the Linkerd proxies use for mTLS. 

So, those proxies will no longer be able to communicate with each other because their certificates will be invalid. The effect is that any workload that is injected with the Linkerd proxy will not be able to communicate with each other until a valid trust anchor is rotated in.
